app/assets/stylesheets/application.scss

```diff
@@ -108,3 +102,17 @@ tr input[type=text] {
   padding: 0 .5rem .5rem;
   font-size: 8px;
 }
+
+.sidebar {
+  margin-top: 135px;
+  height: 100%;
+}
+
+.container {
+  max-width: 1440px;
+}
+
+.brand-logo {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.